internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -1295,7 +1295,13 @@ func (h *Handler) listUserShares(r *http.Request, filters []*collaboration.ListS
 			}
 
 			if statResponse.Status.Code != rpc.Code_CODE_OK {
-				return nil, errors.New("could not stat share target")
+				if statResponse.Status.Code == rpc.Code_CODE_NOT_FOUND {
```

Comment:
Can we also add this in listPublicShares?

Reply:
totally. I'd get hands on it tomorrow morning first thing.

Reply:
Actually giving this a second thought, as this PR has been messy enough with the testing I would like to create a new one for these changes and keep changes as "atomic" as possible. This PR is blocking us from finishing http://github.com/owncloud/ocis/pull/409 and the earlier we merge it  the better 🙄

Reply:
Oh okay. No worries then. I'll merge it